robot1: 1
robot2: 4
arm:
  Return_Delay_Time: 25
  Operating_Mode: 3
  Acceleration_Limit: 9982
  Velocity_Limit: 2900
  Profile_Acceleration: 1500
  Profile_Velocity: 750
  Homing_Offset: -125480
gripper:
  Return_Delay_Time: 25
  Acceleration_Limit: 9982
  Velocity_Limit: 2900
  Profile_Acceleration: 1500
  Profile_Velocity: 750